cmake-tool/helpers/application_settings.cmake

```diff
@@ -49,9 +49,6 @@ function(ApplyData61ElfLoaderSettings kernel_platform kernel_sel4_arch)
     if(KernelPlatformZynqmp AND KernelSel4ArchAarch32)
         set(IMAGE_START_ADDR 0x8000000 CACHE INTERNAL "" FORCE)
     endif()
-    if(KernelPlatformSpike AND KernelSel4ArchRiscV32)
-        set(IMAGE_START_ADDR 0x80400000 CACHE INTERNAL "" FORCE)
```

Comment:
this can be removed because of the DTS overlay change in another PR?

Reply:
Yea, it was being ignored by opensbi before and wasn't having any effect.